finality-aleph/src/network/split.rs

```diff
@@ -75,21 +75,28 @@ async fn forward_or_wait<
     receiver: &Arc<Mutex<R>>,
     left_sender: &mpsc::UnboundedSender<LeftData>,
     right_sender: &mpsc::UnboundedSender<RightData>,
-) {
+) -> bool {
+    trace!(target: "aleph-network", "forward_or_wait called");
```

Comment:
I think I would remove this specific `trace`. The ones below are welcome to stay, but they need a tad better contents, e.g. "Forwarding left/right data." and "Split data channel ended.".

Reply:
Done